lib/kadai-core-test/src/test/java/acceptance/KadaiEngineInitalizationTest.java

```diff
@@ -20,23 +20,35 @@
 
 import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
 
+import io.kadai.KadaiConfiguration;
+import io.kadai.KadaiConfiguration.Builder;
 import io.kadai.common.api.KadaiEngine;
+import io.kadai.common.api.KadaiEngine.ConnectionManagementMode;
 import io.kadai.common.internal.KadaiEngineImpl;
 import io.kadai.spi.priority.api.PriorityServiceProvider;
 import io.kadai.spi.priority.internal.PriorityServiceManager;
 import io.kadai.task.api.models.TaskSummary;
 import io.kadai.testapi.KadaiInject;
 import io.kadai.testapi.KadaiIntegrationTest;
 import io.kadai.testapi.WithServiceProvider;
+import io.kadai.testapi.extensions.TestContainerExtension;
 import java.lang.reflect.Field;
+import java.sql.SQLException;
 import java.util.List;
 import java.util.OptionalInt;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import org.apache.ibatis.transaction.TransactionFactory;
 import org.assertj.core.api.InstanceOfAssertFactories;
 import org.junit.jupiter.api.Nested;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.TestInstance;
 import org.junit.jupiter.api.TestInstance.Lifecycle;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.boot.test.system.CapturedOutput;
+import org.springframework.boot.test.system.OutputCaptureExtension;
 
+@ExtendWith(OutputCaptureExtension.class)
 @KadaiIntegrationTest
 public class KadaiEngineInitalizationTest {
```

Comment:
Can you fix the typo here please :)

`KadaiEngineInitializationTest`

Reply:
Yes I can